src/Core/Resources.resx

```diff
@@ -174,8 +174,8 @@
     <comment>Error when AddIdentity is not called</comment>
   </data>
   <data name="NoTokenProvider" xml:space="preserve">
-    <value>No IUserTokenProvider named '{0}' is registered.</value>
-    <comment>Error when there is no IUserTokenProvider</comment>
+    <value>No IUserTwoFactorTokenProvider named '{0}' is registered.</value>
```

Comment:
Consider using `nameof(T)` by passing in a `{1}` into here.

Reply:
Sure I'll update this instance since I'm editing this, but I don't want to update all the resources that need this change in this PR

Reply:
Works for me.